### DISCUSSION TITLE
Why I'm Not Being Notified of New PRs?

### DISCUSSION THREAD

**Discussion by tajmone** [community/community#21588] [New to GitHub]
<p>I’m always careful to check my GitHub notification multiple times during the day, both in the browser and via email, so I don’t miss out anything that’s happening.</p>
<p>For some reason, I never get notified when someone creates a pull request in one of my repositories, so I end up discovering these PRs after months for repositories which are not very active — I have over 100 repositories, so I can’t possibly manually check all of them.</p>
<p>I looked into my GH account settings for Notifications, and everything is checked except for vulnerabilities reports. I don’t see a specific entry for <em>pull requests</em>, but I’m assuming these fall under the general rule of repositories that I’m following (since I’ve created them I implicitly follow them, right?).</p>
<p>Because I regularly receive an email for every notification that I see in the browser, I exclude that the problem is related to SPAM filters — it’s just that I never seem to get an email for pull requests.</p>
<p>Am I doing something wrong? missed out some settings? or is it just that GitHub doesn’t notify about pull requests in one’s own repositories?</p>
<p>Thanks for the support!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>If it’s just one of your repositories, is it possible you have “unwatched” it in the past? If you click on the :eye: watching icon at the top of the main repository page, you should be able to see a list of people watching the repository - those people will receive notifications.</p>

**Comment by tajmone** [comment]
<aside class="quote group-github-employees" data-username="canuckjacq" data-post="2" data-topic="182880" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 canuckjacq:</div>
<blockquote>
<p>If it’s just one of your repositories, is it possible you have “unwatched” it in the past?</p>
</blockquote>
</aside>
<p>I’m pretty sure I didn’t. But these repositories are quite old (3 or more years), so maybe GitHub changed the default settings in the meantime?</p>
<aside class="quote group-github-employees" data-username="canuckjacq" data-post="2" data-topic="182880" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 canuckjacq:</div>
<blockquote>
<p>If you click on the :eye: watching icon at the top of the main repository page, you should be able to see a list of people watching the repository - those people will receive notifications.</p>
</blockquote>
</aside>
<p>Ahhh! That’s the problem: I wasn’t among the watchers.</p>
<p>Thanks to your tip I discovered that I can use the :eye: tab to change the repository specific notification settings — for some reason, pull requests was disabled.</p>
<p>I’m quite surprised that such an important feature as pull-requests would be disabled by default, after all I’m the repo maintainer and if I miss notifications for PRs I’ll be doing a disservice to my contributors.</p>
<p>Isn’t there a way to configure these notifications globally, to ensure that all my repositories (which are not forks) will auto-notify me for every activity?</p>

**Comment by canuckjacq** [comment]
<p>I’ve checked my non-staff account and “All activity” is selected on all owned repositories.</p>
<p>But it also could be just a matter of settings changing over the years. I’m glad you will get emails from now on!</p>

**Comment by tajmone** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a>, I now think I’ve realized why I wasn’t getting pull requests notifications for some of these repositories: during their preparation stage I had created them as <em>private</em> repositories, and only made them public when they were production ready.</p>
<p>Could it be that private repositories for <strong>Free GH accounts</strong> automatically unset pull request notifications, since they don’t allow third party pull requests (at least that was the case in the past, when no collaborators were allowed for Free private repositories, but Free accounts policy has changed many times over in the past decade)?</p>
<p>If this was the case, making these repositories public wouldn’t automatically enable PR notifications.</p>
<p>Maybe this is something worth mentioning in GH Docs, or showing a pop-up reminder to end users when they switch the repository status between <em>public</em> and <em>private</em>, since it’s easy to forget about these details.</p>

**Comment by Jarod42** [comment]
In https://github.com/settings/notifications
we can manage way to be notified by "type"
- watching
- Participating, @mentions and custom

One (watching) might have been unset, that's why you receive some kind of notification, and not others.
